src/OpenTelemetry.Extensions.Enrichment/TraceEnrichmentServiceCollectionExtensions.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Extensions.DependencyInjection;
 /// <summary>
 /// Extension methods to register telemery enrichers.
 /// </summary>
-public static class OpenTelemetryEnrichmentServiceCollectionExtensions
+public static class TraceEnrichmentServiceCollectionExtensions
```

Comment:
Does it make sense to also add an overload `AddTraceEnricher<T>(this IServiceCollection) where T : TraceEnricher`?

Reply:
There's one with this signature already, or were you meaning to add another param?

Reply:
No params, just a generic method

Reply:
Oh, it seems that I overlooked it. Resolving...